frontend/views/containers/chatroom/ChatMembersAllModal.vue

```diff
@@ -0,0 +1,434 @@
+<template lang='pug'>
+modal-base-template.has-background(ref='modal' :fullscreen='true' :a11yTitle='L("Channel members")')
+  .c-container
+    .c-header
+      div
+        i18n.is-title-2.c-title(
+          tag='h2'
+        ) Members
+        .c-description {{ attributes.name }} ∙ {{attributes.private ? L("Private channel") : L("Public channel")}}
+
+    .card.c-card
+      search(
+        :placeholder='L("Search...")'
+        :label='L("Search")'
+        v-model='searchText'
+      )
+
+      .c-member-count.has-text-1(
+        v-if='searchText && searchCount > 0'
+        data-test='memberSearchCount'
+        v-safe-html='resultsCopy'
+      )
+
+      i18n.c-member-count.has-text-1(
+        v-if='searchText && searchCount === 0'
+        tag='div'
+        :args='{searchTerm: `<strong>${searchText}</strong>`}'
+      ) Sorry, we couldn't find anyone called "{searchTerm}"
+
+      .c-list-to-add
+        .is-subtitle
+          i18n(
+            tag='h3'
+            :args='{  nbMembers: chatRoomUsersInSort.length }'
+          ) Channel members ({nbMembers})
+
+        transition-group(
+          v-if='addedMembers'
+          name='slide-list'
+          tag='ul'
+          data-test='joinedChannelMembersList'
+        )
+          li.c-search-member(
+            v-for='{username, displayName, departedDate} in addedMembers'
+            :key='username'
+          )
+            profile-card(:username='username' direction='top-left')
+              .c-identity
+                avatar-user(:username='username' size='sm')
+                .c-name(data-test='username')
+                  span
+                    strong {{ localizedName(username) }}
+                    .c-display-name(v-if='displayName !== username' data-test='profileName') @{{ username }}
+
+              .c-actions(v-if='isJoined && removable(username)')
+                button.is-icon(
+                  v-if='!departedDate'
+                  :data-test='"removeMember-" + username'
+                  @click.stop='removeMember(username)'
+                )
+                  i.icon-times
+                .has-text-success(v-else)
+                  i.icon-check
+                  i18n Removed.
+                  button.is-unstyled.c-action-undo(
+                    @click.stop='addToChannel(username, true)'
+                  ) Undo
```

Comment:
You've perfectly localized almost every string but missed this one :)

Reply:
Got it